.github/workflows/publish_connectors.yml

```diff
@@ -16,6 +16,16 @@ on:
         description: "Whether to publish as a pre-release (tagged for development) or as a main-release (production)"
         default: pre-release
         type: string
+      publish-java-tars:
+        description: "Whether to publish Java connector tar files."
+        required: false
+        default: false
+        type: boolean
+      original-trigger:
```

Comment:
The workflow behaves differently depending on whether it is triggered by a push to or a workflow dispatch (manually), and we need a way to have the same behavior when using workflow call. For this, we can now specify if a workflow call was triggered by either a push to master or manually.